rider-core/src/test/java/com/github/database/rider/core/PrologTest.java

```diff
@@ -0,0 +1,74 @@
+package com.github.database.rider.core;
+
+import it.unibo.tuprolog.core.Struct;
+import it.unibo.tuprolog.core.TermFormatter;
+import it.unibo.tuprolog.core.parsing.TermParser;
+import it.unibo.tuprolog.solve.*;
+import it.unibo.tuprolog.solve.classic.ClassicSolverFactory;
+import it.unibo.tuprolog.theory.Theory;
+import it.unibo.tuprolog.theory.parsing.ClausesReader;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+
+import static org.junit.Assert.assertTrue;
+
+public class PrologTest {
```

Comment:
I think we can remove this test as its mostly testing the library and no dbrider. We can have unit tests for `PrologAssert` (mentioned in previous comment) using dbunit datasets as input

Reply:
Sure, it was just to illustrate the idea